src/appmixer/mollie/core/ListPayments/component.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "appmixer.mollie.core.ListPayments",
+    "author": "Appmixer <info@appmixer.com>",
+    "description": "Retrieves all payments created with the current website profile.",
```

Comment:
mention and implement the one-page limit 

<img width="841" height="546" alt="Image" src="https://github.com/user-attachments/assets/2e4ddf8b-781e-4802-ae90-9c8d782e2df4" />

Reply:
perhaps the same issue with the ListCustomer and ListPayments